src/Access/AccessControl.cpp

```diff
@@ -763,7 +818,43 @@ std::shared_ptr<const EnabledQuota> AccessControl::getEnabledQuota(
     const String & forwarded_address,
     const String & custom_quota_key) const
 {
-    return quota_cache->getEnabledQuota(user_id, user_name, enabled_roles, address, forwarded_address, custom_quota_key);
+    return quota_cache->getEnabledQuota(user_id, user_name, enabled_roles, address, forwarded_address, custom_quota_key, true);
+}
+
```

Comment:
It seems we don't really need two functions `getAuthenticationQuota()`

Reply:
Removed the unnecessary `getAuthenticationQuota`